index.js

```diff
@@ -125,19 +53,25 @@ xapp.on("error", err => {
   process.exit()
 })
 
-xapp.init(
-  {
-    url: GRAVITY_API_URL,
-    id: GRAVITY_ID,
-    secret: GRAVITY_SECRET,
-  },
-  () => {
-    config.GRAVITY_XAPP_TOKEN = xapp.token
-    if (enableSchemaStitching) {
-      console.warn("[FEATURE] Enabling schema stitching") // eslint-disable-line
-      mergeSchemas().then(startApp)
-    } else {
-      startApp(localSchema)
-    }
+const xappConfig = {
+  url: GRAVITY_API_URL,
+  id: GRAVITY_ID,
+  secret: GRAVITY_SECRET,
+}
+
+xapp.init(xappConfig, () => {
+  config.GRAVITY_XAPP_TOKEN = xapp.token
+
+  if (enableSchemaStitching) {
+    mergeSchemas()
+      .then(schema => {
+        config.SCHEMA = schema
```

Comment:
Due to our need to fetch / merge schemas before we boot MP this had to remain in the root index file and the cleanest way to pass this value over to `src/index` was to set it on config; `mountAndReload` doesn't accept arguments yet but could be something we could update in the future. Open to suggestions as this feels kinda meh...

Reply:
Ah but we do need to reload the schema, because that’s where you would usually make most changes, no? I do think the fetching/merging of schemas could be done after boot, though.

Reply:
Ah, if fetching / merging can happen after then all good 👍 - updated.